src/sections/dataset/publish-dataset/PublishDatasetModal.tsx

```diff
@@ -82,20 +84,15 @@ export function PublishDatasetModal({
         <Stack direction="vertical">
           <PublishDatasetHelpText
             releasedVersionExists={releasedVersionExists}
-            parentCollectionIsReleased={parentCollection.isReleased}
+            nextMajorVersion={nextMajorVersionString}
+            parentCollectionIsReleased={parentCollection.isReleased ?? true}
```

Comment:
`parentCollection.isReleased ?? true`,  this will be set to `true` if the isReleased property is `undefined` or `null`, shouldn't it be `false`?

Reply:
yes, I'm not sure when it would be undefined, but that makes sense